src/main.rs

```diff
@@ -11,6 +11,8 @@ use typstfmt_lib::{format, Config};
 
 const VERSION: &str = env!("TYPSTFMT_VERSION");
 const CONFIG_FILE_NAME: &str = "typstfmt.toml";
+const CONFY_APP_NAME: &str = "typstfmt";
```

Comment:
It's a bit awkward to have two variables for one value, we should use one variable instead maybe `config_base_name`.

Reply:
It's awkward, but they are used for different things. Using the same const for 2 different things will decrease readability. I can add a doc-comment, but it won't change much. Do you think it will be less awkward to use the same value twice (in 3 places, 6 in total)?
```rs
confy::load(CONFY_APP_NAME, GLOBAL_CONFIG_BASE_NAME)
confy::load(CONFIG_BASE_NAME, CONFIG_BASE_NAME)
```

Reply:
I think it's less awkward using the same variable twice, in part because you can click on load to understand what they will be used for, and also because if I have two different named constants I assume their value to be different, but I admit that this is opinionated either way

Reply:
Ok, since both sides have cons, I'll do it your way. Maybe use `APP_NAME` then? I think it would have more meaning this way.

Reply:
Sure :)